scripts/fast-reboot

```diff
@@ -361,6 +361,21 @@ function unload_kernel()
     fi
 }
 
+function save_counters_folder() {
+    debug "Saving counters folder before warmboot..."
+
+    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+	modules=("portstat-0" "dropstat" "pfcstat-0" "queuestat-0" "intfstat-0")
+	for module in ${modules[@]}
+        do
+            statfile="/tmp/$module"
+            if [[ -d $statfile ]]; then
+                cp -rf $statfile /host/
```

Comment:
can you create /host/counters/ folder and move counters under there? Keep everything under /host could make /host messy over time.

Reply:
I have updated the code as you reviewed. Thanks a lot.